build.sbt

```diff
@@ -14,8 +14,7 @@ val scalaTest = "org.scalatest" %% "scalatest" % "3.2.13" % Test
 lazy val dockerSettings = Seq(
   dockerExposedPorts := Seq(8080),
   dockerBaseImage := "eclipse-temurin:11.0.16.1_1-jre-jammy",
-  dockerUsername := Some("mqperf"),
-  dockerRepository := Some("europe-west2-docker.pkg.dev/sml-gke-iac"),
```

Comment:
yes then it uses the default - docker.io; the repository name is below in `Docker / packageName := "mqperf-kafka"` (it will differ for each subproject)

Reply:
Oh, okay. Everything else looks good to me. Merging.